hws/sorted_list/main.c

```diff
@@ -0,0 +1,53 @@
+#include <stdhw.h>
+#include <stdio.h>
+
+int main()
+{
+    LinkedList list = linkedListNew();
+    bool work = true;
```

Comment:
По стайлгайду и вообще по смыслу
```suggestion
    bool isWorking = true;
```
Или вообще `isRunning`